number_guesser.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+
+def main():
+    number_guess()
+
+
+def number_guess():
+    """a function to generate a random number and has the user guess"""
+    test = 0
+    user_num = int(input("Guess a whole number(Non negative) from 1 to 100: \
+                            "))
```

Comment:
I forget why I did spaces, though I know they're bad. 

I think it was because atom was being retarded. - and by that I haven't played with the configuration enough to fix the spacing with atom.